### PR TITLE
fix UI Test failures

### PR DESCRIPTION
**Pull request by npark-mozilla**
- On restarttest by closing the custom tab focus instance

remove existing instance of custom view

### CODE REVIEW

**Comment by npark-mozilla** [comment]
with #2455, I'll try to make a fix for that too.

File: app/src/androidTestWebview/java/org/mozilla/focus/screenshots/SettingsScreenshots.java [hunk 1]
```diff
@@ -41,6 +42,7 @@
 import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
 import static android.support.test.espresso.matcher.ViewMatchers.isEnabled;
 import static android.support.test.espresso.matcher.ViewMatchers.withClassName;
+import static android.support.test.espresso.matcher.ViewMatchers.withContentDescription;
```
Comment (sputnikci) [inline app/src/androidTestWebview/java/org/mozilla/focus/screenshots/SettingsScreenshots.java]: \[Checkstyle\] ERROR: Unused import \- android\.support\.test\.espresso\.matcher\.ViewMatchers\.withContentDescription\.

File: app/src/androidTestWebview/java/org/mozilla/focus/screenshots/SettingsScreenshots.java [hunk 2]
```diff
@@ -117,18 +130,28 @@ public void takeScreenShotsOfSettings() throws Exception {
         assertToolbarMatchesText(R.string.preference_search_remove_title);
         Screengrab.screenshot("SearchEngine_Remove_Search_Engines");
         Espresso.pressBack();
-
+
```
Comment (sputnikci) [inline app/src/androidTestWebview/java/org/mozilla/focus/screenshots/SettingsScreenshots.java]: \[Checkstyle\] ERROR: Trailing whitespace

File: app/src/androidTestWebview/java/org/mozilla/focus/screenshots/SettingsScreenshots.java [hunk 3]
```diff
@@ -42,15 +42,12 @@
...
 import static android.support.test.espresso.matcher.ViewMatchers.withText;
-import static android.support.test.espresso.web.sugar.Web.onWebView;
-import static android.support.test.espresso.web.webdriver.DriverAtoms.findElement;
-import static android.support.test.espresso.web.webdriver.DriverAtoms.webClick;
 import static junit.framework.Assert.assertTrue;
-import static org.hamcrest.Matchers.allOf;
-import static org.hamcrest.Matchers.endsWith;
+import static org.hamcrest.Matchers.containsString;
 import mozilla.components.domains.CustomDomains;
+
```
Comment (sputnikci) [inline app/src/androidTestWebview/java/org/mozilla/focus/screenshots/SettingsScreenshots.java]: \[Checkstyle\] ERROR: Unused import \- android\.support\.test\.espresso\.web\.webdriver\.DriverAtoms\.findElement\.

File: app/src/androidTestWebview/java/org/mozilla/focus/screenshots/SettingsScreenshots.java [hunk 4]
```diff
@@ -42,15 +42,12 @@
...
+import static org.mozilla.focus.fragment.FirstrunFragment.FIRSTRUN_PREF;
```
Comment (sputnikci) [inline app/src/androidTestWebview/java/org/mozilla/focus/screenshots/SettingsScreenshots.java]: \[Checkstyle\] ERROR: Unused import \- android\.support\.test\.espresso\.web\.webdriver\.DriverAtoms\.webClick\.

**Pull request merged**